src/runtime/interop.cs

```diff
@@ -326,37 +343,57 @@ internal class TypeOffset {
         public static int nb_true_divide = 0;
         public static int nb_inplace_floor_divide = 0;
         public static int nb_inplace_true_divide = 0;
-#if (PYTHON25 || PYTHON26 || PYTHON27 || PYTHON32 || PYTHON33 || PYTHON34)
+#if (PYTHON25 || PYTHON26 || PYTHON27 || PYTHON32 || PYTHON33 || PYTHON34 || PYTHON35)
         /* Added in release 2.5 */
         public static int nb_index = 0;
 #endif
+#if (PYTHON35)
+        public static int nb_matrix_multiply;
+        public static int nb_inplace_matrix_multiply;
+#endif
         //} PyNumberMethods;
 //typedef struct {
-        public static int mp_length = 0;
-        public static int mp_subscript = 0;
-        public static int mp_ass_subscript = 0;
-//} PyMappingMethods;
-//typedef struct {
         public static int sq_length = 0;
         public static int sq_concat = 0;
         public static int sq_repeat = 0;
         public static int sq_item = 0;
+#if (PYTHON35)
+        public static int was_sq_slice = 0;
+#else
         public static int sq_slice = 0;
+#endif
         public static int sq_ass_item = 0;
+#if (PYTHON35)
+        public static int was_sq_ass_slice = 0;
+#else
         public static int sq_ass_slice = 0;
+#endif
         public static int sq_contains = 0;
         /* Added in release 2.0 */
         public static int sq_inplace_concat = 0;
         public static int sq_inplace_repeat = 0;
-//} PySequenceMethods;
```

Comment:
That's not how the *object* is laid out in memory though, I think. Check object.h in the python source code.

https://github.com/python/cpython/blob/3e91c558fed0e803369f8d2324f102a042cf04e3/Include/object.h

Take a look around line 465.

Easiest thing would be to check the value in a debugger both on the C side and the C# side (you would need to have a debug build of Python and enable native code debugging in order to do that).

However, given that there were no reported problems previously, I would leave it as it was, unless you have a really good understanding of this code and can demonstrate that it was previously incorrect.